Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -1730,6 +1730,7 @@ def slack_send():
     thread_id = args.get('threadID', '')
     severity = args.get('severity')  # From server
     blocks = args.get('blocks')
+    reply_broadcast = args.get('reply_broadcast')
```

Comment:
The cast is required as the input type is a string.
```suggestion
    reply_broadcast = argToBoolean(args.get('reply_broadcast'))
```